youtrack_sdk/client.py

```diff
@@ -164,6 +164,21 @@ def create_issue(self, *, issue: Issue) -> Issue:
             ),
         )
 
+    def update_issue(self, *, issue_id: str, issue: Issue) -> Issue:
```

Comment:
I completely forgot about `muteUpdateNotifications` argument. You can find it in the API documentation. Could you please add it as well? It's important.

Reply:
`muteUpdateNotifications` argument added!